project.yaml

```diff
@@ -1843,50 +1852,79 @@ Layer:
       <<: *osm2pgsql
       # Include values that are rendered as icon without label to prevent mismatch between icons and labels,
       # see https://github.com/gravitystorm/openstreetmap-carto/pull/1349#issuecomment-77805678
-      table: |-
+      table: |2-
```

Comment:
don't use an explicit number of spaces (`|2-`) but use an implicit instead (`|-`)